lms/static/js/sysadmin/mgmt_commands.js

```diff
@@ -129,11 +132,13 @@
         $inputRow.addClass('form-actions')
         var label = $(document.createElement('label'));
         label.html(arg.display_name);
+        var input = $(document.createElement('input'));
+
         if (arg.required){
             label.append('<span>*</span>')
```

Comment:
"*" does not need to be appended to a span. I just chose span over div so that it wouldn't create a line break. However, I can go ahead and change it to simply append directly into the label by doing label.append('*')